### ISSUE TITLE
incorrect information in README and LICENSE

### ISSUE THREAD

**Issue by pavel-demin** [dk5qh/SDR_Transceiver_Firmware_Stemlab#4]
The firmware that you provide consists of two parts:
 - server code written in C (about 2.5k lines of code)
 - FPGA configuration written in Verilog and Tcl (about 3.5k lines of code)

In the README file you write that my code has been very heavily reworked and many new program parts have been added. However, this is only true for the server code written in C. The FPGA configuration written in Verilog and Tcl is the same as in my repository with just a few lines changed to increase the sample rates.

In the LICENSE file you write that this program contains snippets of code originating from 
https://github.com/pavel-demin/red-pitaya-notes. However, this is only true for server code written in C. This is not true for the FPGA configuration.

So please correct the information in the README and LICENSE files to make it clear that the server code is written by you and the FPGA configuration is written by me and both parts contain a similar amount of code.

**Comment by dk5qh** [comment]
I do not agree to your assumption - which is by no means justified - about the extent using your code. This concerns both the C program and the FPGA programming, which was created in hundreds of hours of synthesis runs.  As for the IP cores I used, neither of us are the authors of these cores. Neither of us claims to own the copyright for the IP cores - right?

Besides that, from a legal point of view, the wording in the license file is not objectionable regardless of any amount of your code used.

I think we have given you the necessary recognition and appreciation for your work for the ham community that you really deserve. Therefore, my definitely final statement is that we will emphasize this once again by another editing the readme in this respect.  Secondly, we are convinced that both elements of our firmware are very different from your code and justify the claim of our copyright.

**Comment by pavel-demin** [comment]
> I do not agree to your assumption - which is by no means justified - about the extent using your code. This concerns both the C program and the FPGA programming, which was created in hundreds of hours of synthesis runs.

The FPGA code you showed me in October was almost an exact copy of the code from my repository with just a few parameters changed here and there. The changes represented less than 5% of the total amount of code.

Is it possible to see the current version of your FPGA code?

> As for the IP cores I used, neither of us are the authors of these cores. Neither of us claims to own the copyright for the IP cores - right?

Wrong. The project contains 19 IP cores from my repository. I'm the author of these IP cores.

> Besides that, from a legal point of view, the wording in the license file is not objectionable regardless of any amount of your code used.

Calling "snippets" the code that represents half of your project is certainly not correct.

> I think we have given you the necessary recognition and appreciation for your work for the ham community that you really deserve.

In the README file it's written that my code is heavily reworked. While it's true for the server code, it's not true for the FPGA code. The FPGA code that you showed me in October was not heavily reworked.

**Comment by pavel-demin** [comment]
One more correction.

The link in the README file points to a repository that doesn't exist anymore. Please change it to the following link:

http://pavel-demin.github.io/red-pitaya-notes/

**Comment by pavel-demin** [comment]
Just to make sure that there is no misunderstanding here are the corrections that I propose to do. 

1. In the LICENSE file, replace

```
In case that this program contains snippets of code originating from
https://github.com/pavel-demin/red-pitaya-notes and these would constitute a 
copyright to Pavel Demin, "The MIT LIcence (MIT)" as listed below shall apply in full.
```

with

```
The parts of the code originating from https://github.com/pavel-demin/red-pitaya-notes constitute a  copyright to Pavel Demin, "The MIT LIcence (MIT)" as listed below shall apply in full.
```

2. In the README file, it would be great if you could write that the C code was heavy reworked and the FPGA code was only slightly modified.

**Comment by pavel-demin** [comment]
I just read the new version of the README file. I find it very unfortunate that you consider my request for a few changes in the README and LICENSE files to be false accusations.

**Comment by dj5er** [comment]
Pavel, we have a term in amateur radio called "Ham-Spirit". Only because of this it was possible to create the topic SDR with all its hard- and software products.  Also your work had a big part in it.

If a member of this amateur radio community makes the work to develop Protocoll 2 for the STEMlab and to completely rebuild the software environment you created years ago, it is altogether a lot of work, a great achievement and for all radio amateurs a great joy.
So far you have not been interested in integrating Protocoll 2 for the STEMlab. Therefore it would have been a sign of inner greatness to accept that someone else can program too.
You should simply accept the software product Protocol 2 for the RedPitaya. 
Unfortunately you do not belong to the amateur radio community and therefore "Ham-Spirit" is probably a foreign word for you.
Thank you for your work in the past, but your current attitude towards radio amateurs seems to be rather hostile, which I am very surprised about.  You don't want to admit it, but with this behaviour you harm all radio amateurs.
with kind regards
heino, dj5er

**Comment by pavel-demin** [comment]
If your "Ham-Spirit" means to not respect copyrights and licenses, then I am happy not to be part of your "Ham-Spirit" community.
